frontend-student/src/components/myCourses/MyCourses.jsx

```
@@ -0,0 +1,258 @@
+    }
+
+    if (error) {
+      return (
+        <div className='bg-red-50 border-l-4 border-red-500 p-4 rounded'>
+          <div className='flex items-center'>
+            <svg
+              className='h-6 w-6 text-red-500 mr-3'
+              fill='none'
+              viewBox='0 0 24 24'
+              stroke='currentColor'
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)